docs/data-sources/administrator_security.md

```diff
@@ -0,0 +1,53 @@
+---
+page_title: "pingone_administrator_security Data Source - terraform-provider-pingone"
+subcategory: "Platform"
+description: |-
+  Data source to retrieve environment administrator sign-on settings.
+---
+
+# pingone_administrator_security (Data Source)
+
+Data source to retrieve environment administrator sign-on settings.
+
+## Example Usage
+
+```terraform
+data "pingone_administrator_security" "administrator_security" {
+  environment_id = var.environment_id
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `environment_id` (String) The ID of the environment that is configured with the security.  Must be a valid PingOne resource ID.  This field is immutable and will trigger a replace plan if changed.
+
+### Read-Only
+
+- `allowed_methods` (Attributes) Indicates the methods to enable or disable for admin sign-on. Required properties are `TOTP` (temporary one-time password), `FIDO2`, or `EMAIL`. (see [below for nested schema](#nestedatt--allowed_methods))
+- `authentication_method` (String) Indicates whether to use PingOne MFA, an external IdP, or a combination of both for admin sign-on. Possible values are `PINGONE`, `EXTERNAL`, or `HYBRID`. The default is `PINGONE`.
+- `has_fido2_capabilities` (Boolean) Indicates whether the environment supports FIDO2 passkeys for MFA.
+- `id` (String) The ID of this resource.
+- `identity_provider` (Attributes) The external IdP, if applicable. (see [below for nested schema](#nestedatt--identity_provider))
```

Comment:
I had to rename these attributes from `provider` to `identity_provider` because `provider` is reserved in terraform.